plugins/modules/content_view_filter.py

```diff
@@ -235,6 +236,12 @@
     'rule_name': {'flat_name': 'name'},
 }
 
+content_filter_rule_deb_spec = {
+    'id': {},
+    'rule_name': {'flat_name': 'name'},
+    'architecture': {},
+}
+
```

Comment:
this is not necessary if you add `deb` to the list in https://github.com/theforeman/foreman-ansible-modules/blob/1727958add1324dd8ee8a0c796806946aed7ae69/plugins/modules/content_view_filter.py#L300

this will enforce the creation of rules via the new `content_view_filter_rule` module, which is the right way ™

Reply:
Okay understood 🙂